example/kustomization.yaml

```diff
@@ -83,3 +83,5 @@ resources:
 # KServe
 - ../contrib/kserve/kserve
 - ../contrib/kserve/models-web-app/overlays/kubeflow
+# BentoML
+- ../contrib/bentoml/bentoml-yatai-stack/default
```

Comment:
This is not a blocker comment, I would like to understand how a third-party component is enabled by default vs. not. What makes bento qualified to be installed by default when other contrib components (other than kserve) are not?

Kserve is a special case as they were developed as part of Kubeflow before graduating out of Kubeflow project. Defining a process here would make it fair for all other contrib components.

Reply:
Yes, that has to be removed for the time being. We will not install it by default.

Reply:
Thanks for the clarification. Removed.